ipyvolume/embed.py

```diff
@@ -1,202 +1,199 @@
-                elif isinstance(value, dict):
-                    for item in value.values():
-                        if isinstance(item, ipywidgets.Widget):
-                            widgets_found.append(item)
-            #print("found", widgets_found)
-            for widgets_found in widgets_found:
-                if widgets_found.model_id in states:
-                    #print("we found that we needed to add ", widget_id, widget)
-                    states[widget.model_id] = widget._get_embed_state(drop_defaults=drop_defaults)
-                    found_new = True
-    return found_new
-import ipywidgets.embed
-
-def save_js(dirname, makedirs=True):
-    """ output the ipyvolume javascript
-    
-    :param dirname: folderpath to output js file to
-    :param makedirs: whether to make the directories if they do not already exist
-    
+    return pyv_filename, three_filename
+
+
+def save_requirejs(folderpath="", version="2.3.4"):
+    """ download and save the require javascript to a local file
+
+    :type folderpath: str
+    :type version: str
     """
-    dir_name_dst = os.path.abspath(dirname)
-    dir_name_src = os.path.join(os.path.abspath(ipyvolume.__path__[0]), "static")
-    if not os.path.exists(dir_name_dst) and makedirs:
-        os.makedirs(dir_name_dst)
-    dst = os.path.join(dir_name_dst, "ipyvolume.js")
-    src = os.path.join(dir_name_src, "index.js")
-    shutil.copy(src, dst)
-
-def embed_html(filename, widgets, makedirs=True, copy_js=True,
-               drop_defaults=False, all=False, title="ipyvolume embed example", external_json=False,
-               indent=2,
-               template=template, template_options={"embed_url":ipywidgets.embed.DEFAULT_EMBED_SCRIPT_URL},
-               widget_view_template=widget_view_template, **kwargs):
-    """ output the the ipywidgets to a standalone html file
-    
-    :param filename: filepath to output to
-    :param widgets: list of ipywidget instances
-    :param makedirs: whether to make directories in the filename path, if they do not already exist
-    :param copy_js: whether to copy a local version of the ipyvolume javascript to the same folder
-    
+    url = "https://cdnjs.cloudflare.com/ajax/libs/require.js/{version}/require.min.js".format(version=version)
+    filename = "require.min.v{0}.js".format(version)
+    filepath = os.path.join(folderpath, filename)
+    download_to_file(url, filepath)
+    return filename
+
+
+def save_embed_js(folderpath="", version=wembed.__html_manager_version__):
+    """ download and save the ipywidgets embedding javascript to a local file
+
+    :type folderpath: str
+    :type version: str
+
     """
+    url = u'https://unpkg.com/@jupyter-widgets/html-manager@{0:s}/dist/embed-amd.js'.format(version)
+    filename = "embed-amd_v{0:s}.js".format(version[1:])
+    filepath = os.path.join(folderpath, filename)
+
+    download_to_file(url, filepath)
+    return filename
+
+
+def save_font_awesome(dirpath='', version="4.7.0"):
```

Comment:
could you mark this as TODO, and link to the original issue on ipywidgets. I'm happy to keep this for now though.